optimum/exporters/onnx/utils.py

```diff
@@ -190,6 +224,22 @@ def _get_submodels_and_onnx_configs(
     preprocessors: list[Any] | None = None,
     model_kwargs: dict | None = None,
 ):
+    if model.config.model_type == "metaclip_2":
```

Comment:
you should guard this with a check if the model is a PreTrainedModel (it can be a diffusion pipelline for example)